src/platforms/windows/daemon/windowsdaemonserver.cpp

```diff
@@ -77,8 +78,19 @@ int WindowsDaemonServer::run(QStringList& tokens) {
 
   WindowsDaemon daemon;
 
-  DaemonLocalServer server(qApp);
-  if (!server.initialize()) {
+  QLocalServer server(qApp);
+  server.setSocketOptions(QLocalServer::WorldAccessOption);
+  connect(&server, &QLocalServer::newConnection, [&]() {
```

Comment:
Similar question to macOS daemon - was this work being done elsewhere before?

Reply:
Yes - see my response above. TLDR: It was being done by `DaemonLocalServer`